CITATION.cff

```diff
@@ -39,6 +39,7 @@ authors:
     given-names: Shantanu
     orcid: https://orcid.org/0000-0003-3150-3025
 title: "Pycytominer: Python package for processing image-based profiling data"
-version: "0.3.0"
+# This version is updated using `cz bump` command
+version: "0.0.0"
```

Comment:
Double checking: should this match the existing version (`0.2.0`)? Or does this act in the same manner as the pyproject.toml Poetry version (where it's solely a placeholder and will not be updated)?

Reply:
Yes. It should match the current version so I fixed it. But it would have been corrected during the cz bump anyways.